addons/udes_stock/models/stock_picking.py

```diff
@@ -983,6 +983,13 @@ def action_assign(self):
         """
             Override action_assign to reserve full packages if applicable
         """
+
+        # This is only called when manually clicking check_availitbily
+        # or called drectly.
+        # This isn't called when next picking is assigned which is done on the
+        # move level
+        # TODO: Check this!
```

Comment:
It's not clear to me what are the consequences of the above.
Is there anything we should to fix / clarify that?
If not, can you improve the message?